invenio_communities/generators.py

```diff
@@ -176,7 +176,7 @@ def query_filter(self, identity=None, **kwargs):
         # Gives access to all community members.
         return Q(
             "terms",
-            **{"id": self.communities(identity)}
+            **{"_id": self.communities(identity)}
```

Comment:
Question: Is this a fix to a bug you've found?

Reply:
Yes I suppose. `"id"` for a community is still the slug, but the CommunityRoleNeed stores the uuid, so the permission search didn't work without this change. What I've noticed was that a `uuid` (literally) field was also present in the document during tests, but I couldn't find its matching official mapping, so I went for `_id` which is guaranteed to be there. It is a little strange given that `_id` is the ES document id, but again I didn't see another official field where the uuid is kept.